### PR TITLE
Add Bulletproofs+ code blog post

### PR DESCRIPTION
**Pull request by SamsungGalaxyPlayer**
Add Bulletproofs+ code blog post

https://gist.github.com/SarangNoether/ee6367fa8b5500120b2a4dbe23b71694

### CODE REVIEW

**Comment by SamsungGalaxyPlayer** [comment]
Tested okay on Firefox Nightly android. Other users report the top image as going off the screen.

File: _posts/2020-12-24-Bulletproofs+-in-Monero.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+### Size
+
+"Regardless of the number of outputs in a transaction, the corresponding Bulletproofs+ range proof is *96 bytes* smaller than a Bulletproofs range proof."
+
+This table shows the reduction in size for the most common transaction types seen on the Monero network.
+
+Spent inputs | New outputs | Current size | New size | Reduction, % smaller
+--- | --- | --- | --- | ---
+1 | 2 | 1.42 kB | 1.33 kB | 6.6%
+2 | 2 | 1.92 kB | 1.83 kB | 5.1%
```
Comment (erciccione) [inline _posts/2020-12-24-Bulletproofs+-in-Monero.md:57]: I would suggest to use these tables less when possible. It's not a very mobile-friendly element.

**Review by erciccione — COMMENTED** [review]
Please use this image. I cropped and shrinked the original.

bulletproofs+

**Comment by erciccione** [comment]
Note that another design for the bulletproofs logo was suggested here: https://www.reddit.com/r/Monero/comments/kk1s4u/faster_than_speeding_bulletproofs_more_powerful/

people seem to like it.

**Comment by SamsungGalaxyPlayer** [comment]
Regarding the image, I opted not to add white behind the left icon. I can use a cropped version.

**Review by erciccione — APPROVED** [review]

**Pull request merged**